### ISSUE TITLE
Allow for normalization choice

### ISSUE THREAD

**Issue by lacan** [MouseLand/cellpose#58]
Hello!

We have a scenario where we want to run cellpose on very large images (>90000 pixels in X and Y). So the usual strategy is to compute the 1% and 99% quantiles on a downsampled version of the image and feeding blocks to the network to return the predictions. 

Because we might get different results if we normalize each block independently, an option to not do normalization during 'eval' would be useful...

Thank you for your time.

**Comment by carsen-stringer** [comment]
you have a way to do the stitching? We output all the flows as well so you could use them to make the masks. 

Actually "turning off normalization" is not documented but it is already implemented. If channels is left empty or set to None, then the image will not be renormalized. So you will want to input your image in the 'correct' format for cellpose in that case (shape (2,Ly,Lx) or (Ly,Lx,2) should work, with second (nuclear) channel set to zeros if it doesn't exist). Note this won't work on the command line, only will work in a notebook.
~~~
X = np.zeros((2,Ly,Lx), dtype=np.float32)
outputs = model.eval(X, channels=None)
~~~

**Comment by lacan** [comment]
Hi and thanks for the info!
> you have a way to do the stitching?
I am using a tool that breaks down the image into tiles and then recovers the individual segmented objects and tries to handle overlaps. what I export is the outlines of the masks as polygons to use in this other software. 

> with second (nuclear) channel set to zeros if it doesn't exist
So if I set the second channel to something else and keep `channels=None` then it will run cellpose on the two channels, correct? this is vague [in the documentation where it says](http://www.cellpose.org/static/docs/api.html#cellpose.models.Cellpose.eval) 
>  Second element of list is the optional nuclear channel (0=none, 1=red, 2=blue, 3=green)
by `0` here it means 'use the channel if it's there, correct?

**Comment by carsen-stringer** [comment]
Yes cellpose will run by default using BOTH channels. So put the nuclear channel as the second channel in the X example above and it will be used.

The channels input is confusing and the description vague, sorry, here's the standard way the channels input works:

1st channel: 
0: average of all channels (now NO 2nd channel can be taken)
1: img[:,:,0]
2: img[:,:,1]
3: img[:,:,2]

2nd channel:
0: set to zero
1: if 1st channel not 0, img[:,:,0]
2: if 1st channel not 0, img[:,:,1]
3: if 1st channel not 0, img[:,:,2]

**Comment by carsen-stringer** [comment]
normalization can now be turned off when running the model in a notebook

**Comment by lacan** [comment]
Merci beaucoup !
All the best

**Comment by lacan** [comment]
Reopening this to request that normalization can be turned off via the CLI as well?
